### DISCUSSION TITLE
简码字词的设置和调整

### DISCUSSION THREAD

**Discussion by forfudan** [forFudan/yuhao#22] [General]
此议题用于讨论一、二、三级简码字词的设置和调整。凡建议改动的，可以标注原因，如：单字频率、手感、简字词联合频率等等。请不要使用“我觉得这个字使用率更高”之类的语句。

**Comment by white1033** [comment]
台灣版的字頻和詞頻或許可以參考
字頻表：https://language.moe.gov.tw/001/Upload/files/SITE_CONTENT/M0001/PIN/biau1.htm?open
詞頻表：https://language.moe.gov.tw/001/Upload/files/SITE_CONTENT/M0001/PIN/biau2.htm?open

**Comment by forfudan** [comment]
> 台灣版的字頻和詞頻或許可以參考 字頻表：https://language.moe.gov.tw/001/Upload/files/SITE_CONTENT/M0001/PIN/biau1.htm?open 詞頻表：https://language.moe.gov.tw/001/Upload/files/SITE_CONTENT/M0001/PIN/biau2.htm?open

謝謝，目前的單字簡碼就是使用這個字頻表的，加上了一些考量手感的調整。詞頻表是好東西，我回頭重排一下。

**Comment by floralattice** [comment]
汇报一个简码设置问题：
鹆 ttl
谷 ttlv
虽然说是不要使用“我觉得这个字使用率更高”之类的语句，但我想这两个字应该还是不必去查字频表了罢……

**Reply by forfudan** [reply]
三級簡碼同字頻無直接關係，因爲其並不節約碼長（都是四碼）。此處「鹆」並非三簡，而是在三碼位上列出了所有的二根字。
